src/Stratis.Bitcoin.Tests/Consensus/ChainedHeaderTreeTest.cs

```diff
@@ -129,31 +131,33 @@ public void SetupCheckpoints(params CheckpointFixture[] checkpoints)
 
                 this.CheckpointsMock
                     .Setup(c => c.GetCheckpoint(It.IsNotIn(checkpoints.Select(h => h.Height))))
-                    .Returns((CheckpointInfo)null);
+                    .Returns((CheckpointInfo) null);
```

Comment:
Also this should be configured because it is not consistent style with what we have elsewhere.